### DISCUSSION TITLE
Error time.sleep(): OSError: [WinError 6] The handle is invalid

### DISCUSSION THREAD

**Discussion by namuetn** [ultrafunkamsterdam/undetected-chromedriver#1351] [General]
![image](https://github.com/ultrafunkamsterdam/undetected-chromedriver/assets/48872422/fc78d696-2fb3-4102-8dbd-67fc664604cd)
When I run python main.py -f answer.txt using undetected_chromedriver when the program ends it always gives an error as shown above. I went to the code base to comment the time.sleep(0.1) segment.
then no more error

**Comment by FayzulSaimun** [comment]
Facing the same issue

**Reply by m0n1serrat** [reply]
такая же проблема
